src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+<html lang="en" class="page">
+  <head >
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page__content">
+    <div class="page__container">
+      <div class="box box--1">1</div>
+      <div class="box box--2">2</div>
+      <div class="box box--3">3</div>
+      <div class="box box--4">4</div>
+      <div class="box box--5">5</div>
+      <div class="box box--6">6</div>
+    </div>
```

Comment:
@DarinaSavytska is correct. Better to make container a block, and boxes as its elements.